src/sdk/types/index.ts

```diff
@@ -386,10 +387,25 @@ export type LocationRequest = {
   useBle?: boolean;
   useDeadReckoning?: boolean;
   useForegroundService?: boolean;
+  foregroundServiceNotificationOptions?: ForegroundServiceNotificationOptions;
   useGps?: boolean;
   useWifi?: boolean;
 };
 
+export type ForegroundServiceNotificationOptions = {
+  title?: string;
+  message?: string;
+  showStopAction?: boolean;
+  stopActionText?: string;
+  tapAction?: ForegroundServiceNotificationsTapAction;
+};
+
+export enum ForegroundServiceNotificationsTapAction {
```

Comment:
Deberíamos de documentar como en flutter este enum https://pub.dev/documentation/situm_flutter/latest/sdk/ForegroundServiceNotificationTapAction.html

Reply:
Ademais o correcto sería que non empezaran con mayúsculas certo ?

Reply:
Coidado con esto, non estou tan seguro. Quizais os de web nos poidan indicar que é o máis idiomático de TypeScript.